Starting subprocess with file monitor
Starting server in PID 20787.
Starting subprocess with file monitor
Starting server in PID 20810.
Starting subprocess with file monitor
Starting server in PID 21140.
Starting subprocess with file monitor
Starting server in PID 10150.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10259.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10366.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10466.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10567.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10662.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10757.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10851.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting server in PID 10949.
Traceback (most recent call last):
  File "/home/ben/crispr/cfront/venv/bin/pserve", line 9, in <module>
    load_entry_point('pyramid==1.4.2', 'console_scripts', 'pserve')()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 50, in main
    return command.run()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 325, in run
    serve()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/pyramid-1.4.2-py2.7.egg/pyramid/scripts/pserve.py", line 315, in serve
    server(app)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/loadwsgi.py", line 189, in server_wrapper
    **context.local_conf)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/PasteDeploy-1.5.0-py2.7.egg/paste/deploy/util.py", line 56, in fix_call
    val = callable(*args, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 21, in serve_paste
    serve(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/__init__.py", line 11, in serve
    server = _server(app, **kw)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 44, in create_server
    return cls(application, map, _start, _sock, _dispatcher, adj)
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 76, in __init__
    self.bind_server_socket()
  File "/home/ben/crispr/cfront/venv/local/lib/python2.7/site-packages/waitress-0.8.5-py2.7.egg/waitress/server.py", line 183, in bind_server_socket
    self.bind((self.adj.host, self.adj.port))
  File "/usr/lib/python2.7/asyncore.py", line 342, in bind
    return self.socket.bind(addr)
  File "/usr/lib/python2.7/socket.py", line 224, in meth
    return getattr(self._sock,name)(*args)
socket.error: [Errno 98] Address already in use
Starting subprocess with file monitor
Starting subprocess with file monitor
-------------------- Restarting --------------------
-------------------- Restarting --------------------
-------------------- Restarting --------------------
-------------------- Restarting --------------------
-------------------- Restarting --------------------
-------------------- Restarting --------------------
